defaults/main.yml

```diff
@@ -271,8 +271,8 @@ postgres_log_rotation_age: 1d
 postgres_log_rotation_size: 10MB
 postgres_syslog_facility: LOCAL0
 postgres_syslog_ident: postgres
-postgres_syslog_sequence_numbers: true
-postgres_syslog_split_messages: true
+# postgres_syslog_sequence_numbers: true
```

Comment:
Why do you want to remove this? Override the var in hostvars if you want to use another value.
Also, just delete the line instead of commenting-out, this is why we have version control.